### ISSUE TITLE
[Adaptive Cards - Image]: Screen reader is conveying the same name for the images present under the 'Image' page. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8675]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/explorer/Image.html
OS Version: 23H2(OS Build 25931.1000)
Browser Version: Edge Dev (118.0.2060.1)
Screen Reader: Narrator

**Pre-Requisite:**
1. Turn on the narrator and turn on the scan mode using 'Caps+Spacebar' key.

**Repro Steps:**
1. Open the above URL on the Edge Dev Browser.
2. Press down arrow key and navigate the screen.
3. Press down arrow key and navigate to the images present under the 'Background Color' and observe the issue.

**Actual Result:**
While navigating to the images present under the 'Background color' in scan mode, screen reader conveys the same name for all images. It is announcing the name as 'Cat' for all the images.

**Expected Result:**
While navigating in scan mode, screen reader should convey the different and unique name for the images present under 'Background'. It should announce as 'Cat with red ribbon' for first image and 'Cat with Yellow ribbon' for the second image.

**User Impact:**
It will impact the screen reader user as they will not be able to know whether they are on same image or different image also, they will not able to differentiate between two images.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

![WCAG4 1 2- Screen Reader is announcing the same name for the images](https://github.com/microsoft/AdaptiveCards/assets/95913748/bb1964bd-295b-4bbd-a1a6-c876f5effda9)

![WCAG1 4 2-Same name is defined for the images](https://github.com/microsoft/AdaptiveCards/assets/95913748/5b6dcd15-ff28-4b7e-849c-1d98d1b2bc0f)

https://github.com/microsoft/AdaptiveCards/assets/95913748/4442b01c-8c76-4bd3-a79b-054f43d69666

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
As per the above mentioned reason, adding A11yTTValidated
